doc/manage-environment-variables.md

```diff
@@ -11,28 +11,25 @@ Environment variables are passed to live environments through Terraform by eithe
      description = "Google Tag Manager preview identifier"
    }
    ```
-1. Inject the new variables to the app and/or worker (if in doubt add to both):
+1. Inject the new variable to the app and/or worker (if in doubt add to both):
    ```
    resource "cloudfoundry_app" "beis-roda-app" {
      environment = {
        …
        "GOOGLE_TAG_MANAGER_CONTAINER_ID" = var.google_tag_manager_container_id
        …
    ```
-1. Reference the new variable in our `deploy-terraform.sh` script in **2 places**, one for each environment. Changing the prefix to match the environment name that the variable should be applied to:
-   ```
-   if [ "$TRAVIS_BRANCH" = master ]
-     export TF_VAR_google_tag_manager_environment_preview="$PROD_GOOGLE_TAG_MANAGER_ENVIRONMENT_PREVIEW"
-   if [ "$TRAVIS_BRANCH" = develop ]
-     export TF_VAR_google_tag_manager_environment_preview="$STAGING_GOOGLE_TAG_MANAGER_ENVIRONMENT_PREVIEW"
-   ```
-1. Add 2 new environment variables to Github matching those names from the previous step. [Github settings can be managed here.](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/settings/secrets/actions)
-1. Once all steps are complete and merged, the next time Travis deploys either staging and production those environment variables will be made available to the app
-1. Add those variables to our environment file in the RODA 1Password vault as they cannot be read back out of Travis
+1. Add 2 new environment variables to GitHub - one for staging, one for prod. [GitHub settings can be managed here.](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/settings/secrets/actions)
+  ```
+    STAGING_GOOGLE_TAG_MANAGER_CONTAINER_ID=...
+    PROD_GOOGLE_TAG_MANAGER_CONTAINER_ID=...
+  ```
+1. Once all steps are complete and merged, the next time GitHub Actions deploys either staging and production those environment variables will be made available to the app
```

Comment:
Those `PROD_` and `STAGING_` prefixes aren't magic, so it looks like any new environment variables also need to be referenced in these files too:

- `.github/workflows/deploy.yml`
- `terraform/variables.tf`
- `terraform/app.tf`

Reply:
Ah, thanks, the previous instructions made it seem like the prefixes were indeed magic :O

Reply:
@mec I think you still need to add something about adding the environment variables to `terraform/variables.tf` (?)

Reply:
Sorry, is that not step 1?

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/manage-environment-variables.md

Reply:
Ah... it wasn't showing in the diff 🤦